week 1/lists_n_loops/driver.py

```diff
@@ -0,0 +1,77 @@
+import index_search as test
+import time
+import matplotlib.pyplot as plt
+import random as rnd
```

Comment:
the import order is incorrect. it should be `first party packages, followed by third party, and finally the local packages`